shard-operator/pom.xml

```diff
@@ -121,6 +125,7 @@
         <version>${surefire-plugin.version}</version>
         <configuration>
           <systemPropertyVariables>
+          	<container.image.keycloak>${container.image.keycloak}</container.image.keycloak>
```

Comment:
What is this used for? Maybe a leftover?

Reply:
This is need for KeycloakResource test class to work